gdk/wizard/commons/model.py

```diff
@@ -0,0 +1,122 @@
+from gdk.wizard.commons.fields import Fields
+
+
+class Wizard_model:
+    def __init__(self, data):
+        self.data = data
+        self.field_map = self.data.field_map
+        self.project_config = self.data.project_config
+        self.component_name = self.data.component_name
+
+    def get_author(self):
+        try:
+            return self.project_config[self.component_name][Fields.AUTHOR.value.key]
```

Comment:
Use `return self.project_config.get(self.component_name, default_value)` instead, here and at every other place,

Reply:
gotcha, I'm assuming that you mean that I should replace  return self.project_config[self.component_name] with return self.project_config.get(self.component_name, default_value) and have a default value for component_name ?